ledger/accountdb.go

```diff
@@ -2515,6 +2516,11 @@ func onlineAccountsInitDbQueries(r db.Queryable) (*onlineAccountsDbQueries, erro
 	if err != nil {
 		return nil, err
 	}
+
+	qs.lookupOnlineTotalsHistoryStmt, err = r.Prepare("SELECT data FROM onlineroundparamstail WHERE rnd=?")
```

Comment:
nit: "lookupOnlineHistoryStmt" meant the entire history (for all rounds) so I would have named this lookupOnlineTotalsStmt or lookupOnlineRoundParamStmt (since you are looking up the whole params for the round)